instagram/src/components/PostsContainer/Post.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import HeaderSection from "./HeaderSection";
+import MediaSection from "../MediaContainer/MediaSection";
+import CommentSection from "../CommentContainer/CommentSection";
+import "./Post.css";
+import LikeSection from "./LikeSection";
+
+class Post extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      likes: props.post.likes
+    };
+  }
+
+  addLike = () => {
+    this.setState({ likes: this.state.likes + 1 });
```

Comment:
if you notice this gets a little funky you may wanna try
```suggestion
this.setState(prevState => {
     return {
                likes: prevState.likes + 1
    }
})

Reply:
Thanks, fixed. Your suggestion improves the readability for me